pkg/kubectl/cmd/scale.go

```diff
@@ -227,8 +267,14 @@ func (o *ScaleOptions) RunScale(f cmdutil.Factory, out, errOut io.Writer, cmd *c
 		}
 
 		counter++
-		cmdutil.PrintSuccess(shortOutput, out, info.Object, false, "scaled")
-		return nil
+		internalObj, err := legacyscheme.Scheme.ConvertToVersion(info.Object, info.Mapping.GroupVersionKind.GroupKind().WithVersion(runtime.APIVersionInternal).GroupVersion())
+		if err != nil {
+			// if there's an error, try to print what you have (mirrors old behavior).
+			glog.V(1).Info(err)
+			return o.PrintObj(info.Object, o.Out)
+
+		}
+		return o.PrintObj(internalObj, o.Out)
```

Comment:
this won't work.  You can't print an internal to json or yaml.

Reply:
Lemme check that again...

Reply:
It was wrong in the original PR, I've changed it to `info.AsVersioned()` as we do it in all the places.